Technical Edit of Baldy Release (#1570)
cntt-n/CNTT: doc/gov/chapters/chapter02.md

<a name="2.1"></a>
## 2.1 Introduction

> As an open-source community organization, CNTT has developed the following process to ensure delivery on the CNTT objectives. This process has several goals. First, to inform new participants with the background and overview of this open-source initiative. Second, to communicate the organizational and legal structures under which the community operates. And third, to help new participants to quickly become fully engaged in the CNTT work and to be contributing effectively to the work streams they are interested in.

Reviewer: Should "... to inform new participants with the background ..." be "... to inform new participants about the background ..."

> As an open-source community organization, CNTT has developed the following process to ensure delivery on the CNTT objectives. This process has several goals. First, to inform new participants with the background and overview of this open-source initiative. Second, to communicate the organizational and legal structures under which the community operates. And third, to help new participants to quickly become fully engaged in the CNTT work and to be contributing effectively to the work streams they are interested in.

Reviewer: ```suggestion
As an open-source community organization, CNTT has developed the following process to ensure delivery on the CNTT objectives. This process has several goals. First, to inform new participants about the background and overview of this open-source initiative. Second, to communicate the organisational and legal structures under which the community operates. And third, to help new participants quickly become fully engaged in the CNTT work and begin contributing effectively to the work streams they are interested in.
```

<a name="2.2"></a>
## 2.2 Request Onboarding Info Packet

[Sample Welcome Message Here] 
Welcome to the CNTT group –

> It is wonderful that [Company and person] has decided to join in the effort to make building the infrastructure to support Telecom workloads more robust and standardized. We believe that these efforts will benefit not only the operators, but also the vendors, and ultimately all our customers as well, by hopefully making it easier and simpler to incorporate new technology into our systems.

Reviewer: should we make this personal as in "It is wonderful that you have decided to join ..." -- the salutation should have the name of the person and organisation.

Also, should the first sentence reference "common infrastructure reference platform"

Can we modify the second sentence, "We believe that these efforts will benefit the operators and vendors, and ultimately our customers, by making it easier and simpler to incorporate new technology into our systems."

Reviewer 2: I believe those has been proposed as commit proposals.

> To get you started, I encourage you to go to our Getting Started page to download a packet of information on how to get engaged at the working level with CNTT.  We also have materials in the Wiki and Github repositories, so I hope that all this doesn’t overwhelm you.  We are excited that you will be able to become more engaged in our efforts and as always, we will be happy to help you get up to speed.  

Reviewer: can we have links to "Getting Started", "wiki" and "Github"

The last sentence seems condescending -- maybe a different sentence at the end of the letter "We are excited that you have decided to become engaged with our community and look forward to your active participation" or some such sentiments

> To get you started, I encourage you to go to our Getting Started page to download a packet of information on how to get engaged at the working level with CNTT.  We also have materials in the Wiki and Github repositories, so I hope that all this doesn’t overwhelm you.  We are excited that you will be able to become more engaged in our efforts and as always, we will be happy to help you get up to speed.  

Reviewer: ```suggestion
To get you started, I encourage you to go to our Getting Started page to download a packet of information on how to get engaged at the working level with CNTT.  We also have materials in the Wiki and Github repositories, so I hope that all this doesn’t overwhelm you.  We are excited that you have decided to become engaged with our community and look forward to your active participation.  
```

In addition to the many on-line work streams and workstream related on-line meetings, we also have periodic face to face workshops, which we find are invaluable for speeding the delivery of CNTT’s desired outcomes. More details on upcoming events can be found on the CNTT wiki. 

Feel free to reach out at any time, I am sure we will be more than happy to answer any questions you might have. Welcome again and looking forward to working with you.

- **Guide To Getting Started**

  - **How do I start the process of becoming a participant?** Onboarding starts as soon as you submit a request to join CNTT via email to cntt-gov@lists.opnfv.org. You will receive additional information such as links to the CNTT Getting Started wiki and additional onboarding materials.
  
  - **How long will it take to onboard?** The onboarding process will initially take approximately one hour.
  

>   - **What will I learn during onboarding process?** There will be an overview of the CNTT mission and goals, community organizational structures and processes, as well as an introduction to the technology development status and roadmaps. Additionally, participant responsibilities and legal obligations with be covered. Finally, you will be introduced to the support and logistical information needed to participate fully in the CNTT community.

Reviewer: typo: "... during onboarding process?" --> "... during the onboarding process?"

Reviewer 2: done

>   - **What will I learn during onboarding process?** There will be an overview of the CNTT mission and goals, community organizational structures and processes, as well as an introduction to the technology development status and roadmaps. Additionally, participant responsibilities and legal obligations with be covered. Finally, you will be introduced to the support and logistical information needed to participate fully in the CNTT community.

Reviewer: ```suggestion
  - **What will I learn during the onboarding process?** There will be an overview of the CNTT mission and goals, community organisational structures and processes, as well as an introduction to the technology development status and roadmaps. Additionally, participant responsibilities and legal obligations with be covered. Finally, you will be introduced to the support and logistical information needed to participate fully in the CNTT community.
```
